src/SourceBuild/content/eng/PublishSourceBuild.props

```diff
@@ -87,9 +82,17 @@
           DestinationFolder="$(SourceBuiltLayoutDir)$(PreviouslySourceBuiltReferencePackagesDirName)"
           UseSymbolicLinksIfPossible="true" />
 
+    <Exec Command="git rev-parse HEAD"
```

Comment:
Isn't there a "no git" requirement for SB partners?

Reply:
@dotnet/source-build

Reply:
I'll probably have to do some sort of trick like we do with product versions then to get this to work.

I'll give that a shot.

Reply:
That's correct.  You should not assume that distro maintainers are building the src within the context of a git repo.  The [build command supports building outside a git repo](https://github.com/dotnet/dotnet/blob/main/build.sh#L32).